apps/daffio/src/index.html

```diff
@@ -8,6 +8,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
   <link rel="icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" href="https://use.typekit.net/kqu7meg.css">
```

Comment:
We need to address font loading in a new issue. This can be significantly impactful on load times.

Reply:
Is the font loading too slow? I haven't updated the foundation-demo font to this new one yet, so there are two font stacks running.

Reply:
Yes and no. It's more about how loading the font prevents DOM rendering. Basically, we need to be using: https://github.com/typekit/webfontloader. Can you open a new issue for this?

Reply:
Issue opened: #135